From c91ffd4011731892cc0ba9617dab6d0219c08baf Mon Sep 17 00:00:00 2001
From: Ram Dayal <ram.dayal@st.com>
Date: Thu, 23 Jan 2025 11:09:19 +0100
Subject: [PATCH] arm64: configs: add CONFIG_TYPEC_ADC in fragment-02

This will enable ADC based Type-C driver to monitor
current on usb typec port.

Change-Id: Ia8f8fa8a0eaf38b2288a14f955a49408e22a278f
Signed-off-by: Ram Dayal <ram.dayal@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/430953
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 401331dd0cd1..90567539507d 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -119,6 +119,7 @@ CONFIG_SYSCON_REBOOT_MODE=m
 CONFIG_TEE_REMOTEPROC=y
 CONFIG_TOUCHSCREEN_GOODIX=m
 CONFIG_TOUCHSCREEN_ILI210X=m
+CONFIG_TYPEC_ADC=m
 CONFIG_TYPEC_UCSI=m
 CONFIG_UBIFS_FS=y
 CONFIG_UCSI_STM32G0=m
-- 
2.50.1

